src/app.js

```diff
@@ -100,6 +101,8 @@ async function run(config) {
 
   const poll_period_ms = 10;
 
+  await createStashAccounts.createStashAccounts(config.apis[api_idx]);
```

Comment:
It's using the first node to create stash accounts, which is acceptable for now!

Reply:
This shouldn't be a huge issue - I scripted multiple connections to speed up Steve funding - if we want to make hundreds of stash accounts later on, then we could consider multi-node connections